src/ensembl/src/content/app/custom-download/containers/content/preview-card/PreviewCard.scss

```diff
@@ -0,0 +1,33 @@
+@import 'src/styles/common';
+
+.resultCard {
+  min-height: 75px;
+  padding: 10px 5px;
+  background: $white;
+  margin-bottom: 5px;
+  position: relative;
+  display: block;
+
+  .resultLine {
+    width: 100%;
+    position: relative;
+    clear: both;
+    .lineHeader {
+      text-align: right;
+      color: $dark-grey;
+      width: 35%;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      float: left;
```

Comment:
![image](https://user-images.githubusercontent.com/6834224/66819167-68617280-ef36-11e9-85a6-0ece1e8ba295.png)

It looks like a table. Perhaps instead of `float:left`, and `clear: both`, either the `table` markup or the `display: grid` will be more appropriate?

Reply:
Updated it to use table.